edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.Test/DockerConfigTest.cs

```diff
@@ -142,15 +143,87 @@ public void TestDeserializationError()
         [Fact]
         public void TestSerialization()
         {
-            var createOptions = @"{""Env"": [""k1=v1"", ""k2=v2"", ""k3=v3""], ""HostConfig"": {""PortBindings"": {""43/udp"": [" +
+            string createOptions = @"{""Env"": [""k1=v1"", ""k2=v2"", ""k3=v3""], ""HostConfig"": {""PortBindings"": {""43/udp"": [" +
```

Comment:
Leaving var is acceptable here because the type is pretty obvious when plain initialization is happening